src/src/navigator.h

```diff
@@ -11,18 +11,40 @@
  */
 
 class Navigator {
+private:
+    //Member Data
+    Driver driver;
+    GPS directions;
+    Maze maze;
+    Coordinate destination;
```

Comment:
I don't think this needs to be a member variable. Just make a new variable inside a function any time your need it.